static/docs/command-reference/checkout.md

```diff
@@ -95,6 +95,10 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - recreates links or copies for all checked out files even ones
+  with unchanged checksums. This ensures that link types of all the files in a
+  workspace match configured `cache.type`.
```

Comment:
- `--relink` - recreates links or copies for all checked out files even unchanged ones. This ensures the link types of all the files in the workspace match the project's [`cache.type`] (/doc/command-reference/config#cache).

Reply:
See also iterative/dvc/issues/2983.

Reply:
Autowrapped by precommit hook, I can't change it. 

The text is tailored to whatever we have above. There are two mentions of checksums before, here we refer to them again. So not "unchanged", but "unchanged checksums".

Reply:
I meant that the paragraph I suggested above may not be wrapped properly.

I see no need to talk about checksums in this paragraph. The point is that unchanged files also get relinked, no?

Reply:
Unchanged and unchanged checksums are different, e.g. a copy and a symlink may have the same checksum, but are different things so might be considered changed. We never check whether a file/dir is changed, only if checksum is changed, this is mentioned a few times before in the text. 

Talking about relinking unchanged files will be both incorrect and confusing, i.e. why do I ever need to relink whatever is unchanged? So we don't talk about unchanged files anywhere on this page, we say that we checkout outputs that have checksums not matching their stage desc, i.e. changed, and `checkout --relink` checkouts outputs with matching/unchanged checksums.

Reply:
Hmm I see what you mean @Suor. I think the sentence you're mainly referring to is:

> For any data files where the checksum doesn't match their DVC-file entry, the data file is restored from the cache. 

So maybe the text in question would be best as `...for all data files in the workspace, including the ones normally checked out by this command, as well as existing ones that already have matching checksums in current DVC-files`

```
- `--relink` - recreates links or copies from cache to workspace
  for all data files in the workspace, including the ones normally
  checked out by this command, as well as existing ones that
  already have matching checksums in current DVC-files. This
  ensures the link types of all the files in the workspace match
  the project's [`cache.type`](/doc/command-reference/config#cache).
```
(Please check wrapping.)

This has prompted me to review the language in the description above, BTW, but I will do that in a separate PR. Question: Does `dvc checkout` remove data files no longer referenced in workspace DVC-files? Thanks

Reply:
I applied my suggestion in 0d7e4f8d. Sorry @Suor, hijacking your branch to apply some other changes (including https://github.com/iterative/dvc.org/pull/864#issuecomment-567345548) and move things along 😬

Reply:
p.s. please review the new text and mark this resolved if you agree.

Reply:
Resolving in favor of https://github.com/iterative/dvc.org/pull/864#issuecomment-569373439